tests/web/deprecated/Computer_SoftwareLicense.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Data files for tests are usually in their own directory (emails-tests, fixtures, ...)